__tests__/api.test.js

```diff
@@ -75,15 +75,13 @@ describe('Articles', ()=>{
                 });
             });
     })
-    describe.only('GET /api/articles', ()=>{
+    describe('GET /api/articles', ()=>{
         test("returns an array of article objects with correct properties", ()=>{
             return request(app)
             .get("/api/articles")
             .expect(200)
             .then((response) => {
                 const articles = response.body.articles;
-                expect(articles.length).toBe(13);
```

Comment:
This should not have been deleted as you are running a forEach afterwards. It's important to check for the length of an data we are testing with a forEach so that we can avoid false positives.